runtime/lua/vim/lsp/inlay_hint.lua

```diff
@@ -17,10 +23,12 @@ local augroup = api.nvim_create_augroup('vim_lsp_inlayhint', {})
 
 --- |lsp-handler| for the method `textDocument/inlayHint`
 --- Store hints for a specific buffer and client
+---@param err table?
 ---@param result lsp.InlayHint[]?
 ---@param ctx lsp.HandlerContext
+---@param config vim.lsp.inlay_hint.config?
 ---@private
-function M.on_inlayhint(err, result, ctx, _)
+function M.on_inlayhint(err, result, ctx, config)
```

Comment:
`on_inlayhint` is not mentioned in `:help lsp`, I wonder if we need a list of these handlers near the list at `:help lsp-method`. Otherwise they are much less discoverable.

Reply:
Right, that's because the function is marked as private. I was also unsure of how to document this, since it's being used in places not processed by luadoc rn...